AggregateMetrics/AggregateMetrics/AzureWebApp/TimeDependentCounter.cs

```diff
@@ -0,0 +1,12 @@
+﻿using Microsoft.ApplicationInsights.DataContracts;
+using System;
+
+namespace Microsoft.ApplicationInsights.Extensibility.AggregateMetrics.AzureWebApp
+{
+    public struct TimeDependentCounter
```

Comment:
Agreed, it is a good idea to make new code StyleCop compliant, a big part of which is full documentation comments.